Sources/CasePaths/Never+CasePathable.swift

```diff
@@ -16,21 +16,27 @@ extension Case where Value: CasePathable {
   /// This property can chain any case path into a `Never` value, which, as an uninhabited type,
   /// cannot be embedded nor extracted from an enum.
   public var never: Case<Never> {
-    @Sendable func absurd<T>(_: Never) -> T {}
-    return Case<Never>(embed: absurd, extract: { (_: Value) in nil })
+    get {
+      @Sendable func absurd<T>(_: Never) -> T {}
+      return Case<Never>(embed: absurd, extract: { (_: Value) in nil })
+    }
+    set {}
```

Comment:
These sets are to preserve case key path functionality. Luckily the weirdness is all hidden away in the "internal" `Case` type.